_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+Sound-
+Choose your output device:
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+Select VoiceMeeter Input (VB-Audio Voicemeeter VAIO) as the default
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+After setting VB-Audio Voicemeeter VAIO as our default output device pull up Voicemeeter Banana. In the middle under the Voicemeeter VAIO section we should see the audio levels rising and lowering if you play sounds on our computer. Do you? Great! If not? Try again!
+
+You may or may not be able to hear these right away, if not just take a look in the column down at the A1, A2, A3 and select the ones for the headphones. If you set up along with the tutorial it should be A1. At this point you should be able to hear the audio going through your headphones which will show in the bottom right level bars under the A1 column.
+
+Bottom right Hardware Outputs audio with audio pushed through.
+
+You will notice this has a bunch of different options, including a mono toggle and a mute toggle! At this point you should be hearing your computer audio going through and being pushed to your headphones.
+
+
+### Hardware inputs and microphone
+Are you ready to set up a microphone? I am! Alright, so microphones. The far left section, Hardware Inputs, there should be a 'Select your device' listing, click this and you should get an option between a bunch of different things. Select the WDM version of your microphone for now, we can get into different versions of a device driver later but for now just use WDM.
+
+Picture of a microphone selected in Hardware Inputs section:
+
+You will now notice if your microphone is unmuted it should be moving the audio bar under that device listing. If you want to hear what is going through the microphone you can push the A corresponding to your headphones, if you set them up following this tutorial then they should be A1. Can you hear yourself? Good! If not, something might be wrong. if you see no audio being bounced in the level meter alongside your hardware input, windows audio settings might be weird and you may have to adjust your audio settings within the audio driver for that device in Windows Recording Devices.
+
+You may have noticed we haven't done anything with A2 or A3 yet. If you followed the tutorial we will have set A2 Hardware Output to a Virtual Audio Device. If you select A2 on the Voicemeeter VAIO input, and the Microphone input, you will see down in the right corner the audio level bar for A2 moving. The reason we set up this Virtual Audio Devices is to pull into OBS.
+
+Picture of Audio bars moving after A1 and A2 set:
+
+
+### OBS Audio and alerts
+Lets get our audio into OBS! Open OBS and go to Settings → Audio. For this all we do is set Desktop Audio to Cable Input (VB-Audio Virtual Cable). Disable everything else since we are using Voicemeeter Banana to route and mix all our audio to one place.
```

Comment:
Double check that this is supposed to be "Cable Input" and not "Cable Output". It gets a little confusing when I'm reading through this stuff and don't have a computer ready to work through this stuff. I swear after detox I'll go through this explicitly step-by-step.

Reply:
Double checked, default installed driver is called Cable Input. It is a little weird but I left the naming as is for all of the virtual audio drivers that get installed/used so that it would be less confusing. If I renamed them and told the reader how to rename, if they accidentally renamed it wrong then there is even more confusion.

Reply:
we can wait to resolve this until you get a chance to double check though if you want lol